tests/isolation2/sql/test_dropped_table.sql

```diff
@@ -0,0 +1,28 @@
+-- Ensure diskquota does not save information about dropped table during restart cluster by invalidates it at startup
+
+!\retcode gpconfig -c diskquota.naptime -v 2 --skipvalidation;
+!\retcode gpstop -u;
+
+1: CREATE SCHEMA dropped_schema;
+1: SET search_path TO dropped_schema;
+1: SELECT diskquota.set_schema_quota('dropped_schema', '1 MB');
+1: SELECT diskquota.wait_for_worker_new_epoch();
+1: CREATE TABLE dropped_table(id int) DISTRIBUTED BY (id);
+1: INSERT INTO dropped_table SELECT generate_series(1, 100000);
```

Comment:
```suggestion
1: INSERT INTO dropped_table SELECT generate_series(1, 10000);
```
To avoid flaky tests, please make table smaller.